reference/conanfile/tools/intel.rst

```diff
@@ -0,0 +1,123 @@
+.. _conan_tools_intel:
+
+
+conan.tools.intel
+===================
+
+This tool helps you to manage the new Intel oneAPI `DPC++/C++ <https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top.html>`_ and
+`Classic <https://software.intel.com/content/www/us/en/develop/documentation/cpp-compiler-developer-guide-and-reference/top.html>`_ ecosystem in Conan.
+
+.. warning::
+
+    These tools are **experimental** and subject to breaking changes.
+
+.. warning::
+
+    macOS* is not supported for the Intel oneAPI DPC++/C++ (icx/icpx or dpcpp) compilers. For macOS or Xcode* support, you'll have to use the Intel C++ Classic Compiler.
+
+
+IntelCC
+=========
+
+Available since: `1.41.0 <https://github.com/conan-io/conan/releases>`_
+
+
+.. note::
+
+    Remember, you need to have installed previously the `Intel oneAPI software <https://software.intel.com/content/www/us/en/develop/tools/oneapi/all-toolkits.html#gs.cgeofk>`_.
+
+
+This tool brings the mechanism to load the Intel oneAPI environment variables loaded by the Intel script ``setvars.sh|bat`` and use the different compilers internally in your
+project. That script is the first step to start using the Intel compilers because it's setting some important variables in your local environment.
+Basically, what ``IntelCC`` is doing internally:
```

Comment:
```suggestion
That script is the first step to start using the Intel compilers because it's setting some important variables in your local environment.
The``IntelCC`` generator:
```